source/frontend/src/features/mapSideBar/acquisition/tabs/stakeholders/detail/StakeHolderContainer.test.tsx

```diff
@@ -83,101 +78,4 @@ describe('StakeHolderContainer component', () => {
     const { asFragment } = setup({});
     expect(asFragment()).toMatchSnapshot();
   });
-
-  it('groups multiple interest holder properties by acquisition file id', async () => {
```

Comment:
?

Reply:
All the tests for the grouping logic was movied into its own test file that only deals with that.